.version

```diff
@@ -1 +1 @@
-1.9.7
+1.9.8
```

Comment:
We published 1.9.8 in 1.2alpha13, can you rebase this and bump to 1.9.9 @johren-hpe ?

Reply:
Done